packages/kernel-ui/src/components/VatTable.tsx

```diff
@@ -12,48 +23,114 @@ export const VatTable: React.FC<{
   }
 
   return (
-    <div className={`${styles.table} ${styles.subclusterTable}`}>
-      <table data-testid="vat-table">
+    <Box className="w-full mt-4">
+      <table
+        data-testid="vat-table"
+        className="w-full border-collapse border-t border-muted"
+      >
         <thead>
-          <tr>
-            <th>ID</th>
-            <th>Source</th>
-            <th>Parameters</th>
-            <th>Actions</th>
+          <tr className="border-b border-muted">
+            <th className="text-left py-2 px-3 border-r border-muted">
+              <TextComponent
+                variant={TextVariant.BodyXs}
+                fontWeight={FontWeight.Medium}
+                color={TextColor.TextMuted}
+              >
+                ID
+              </TextComponent>
+            </th>
```

Comment:
It is a lot of repetition
```suggestion
            <TableHeadItem classStyle="border-r border-muted">ID</TableHeadItem>
```

Reply:
It didn't seem to me but sure, I can do this :)